src/pages/Organization/components/OrganizationSelector.tsx

```diff
@@ -1,6 +1,180 @@
+// import { useQuery } from "@tanstack/react-query";
```

Comment:
Remove the commented out code. You can always get the old code using git commands like git checkout and git reset.